### PR TITLE
Aclfromhell

### PR DESCRIPTION
**Pull request by elesbom**

### CODE REVIEW

File: dbaas/drivers/mongodb.py [hunk 1]
```diff
@@ -522,6 +526,10 @@ def remove_metric_collector_user(self, username):
 
 class MongoDBReplicaSet(MongoDB):
 
+    @property
```
Comment (ricardosdias) [inline dbaas/drivers/mongodb.py]: Property is defined on MongoDB

File: dbaas/drivers/mysqldb.py [hunk 2]
```diff
@@ -463,6 +466,11 @@ def remove_metric_collector_user(self, username):
 
 
 class MySQLFOXHA(MySQL):
+
+    @property
```
Comment (ricardosdias) [inline dbaas/drivers/mysqldb.py]: Property is defined on MySQL

File: dbaas/tsuru/views.py [hunk 3]
```diff
@@ -576,6 +484,17 @@ def check_database_status(database_name, env):
         return log_and_response(
             msg=msg, http_status=status.HTTP_412_PRECONDITION_FAILED)
 
+    task = TaskHistory.objects.filter(
+        arguments__contains="Database: {}, Environment: {}".format(
+            database_name, env
+        ), task_status="ERROR")
+
```
Comment (ricardosdias) [inline dbaas/tsuru/views.py]: This filter will get any task for this database when status = ERROR

File: dbaas/drivers/mongodb.py (hunk 1 above)
Comment (elesbom) [reply dbaas/drivers/mongodb.py]: ill remove then

File: dbaas/drivers/mysqldb.py (hunk 2 above)
Comment (elesbom) [reply dbaas/drivers/mysqldb.py]: ill remove then

File: dbaas/tsuru/views.py (hunk 3 above)
Comment (elesbom) [reply dbaas/tsuru/views.py]: U right. We will back later to refactor all legacy method ok ?

File: dbaas/tsuru/views.py [hunk 4]
```diff
@@ -185,12 +209,15 @@ def post(self, request, database_name, format=None):
     def delete(self, request, database_name, format=None):
         env = get_url_env(request)
         data = request.DATA
-        LOG.debug("Request DATA {}".format(data))
+        LOG.debug("Tsuru Unbind App DELETE Request DATA {}".format(data))
 
         response = check_database_status(database_name, env)
-        if type(response) != Database:
+        if not isinstance(response, Database):
```
Comment (otherpirate) [inline dbaas/tsuru/views.py]: You can use self.model instead of Database

**Pull request merged**